src/coreclr/tools/aot/ILCompiler/Program.cs

```diff
@@ -184,6 +186,7 @@ private ArgumentSyntax ParseCommandLine(string[] args)
                 syntax.DefineOption("waitfordebugger", ref waitForDebugger, "Pause to give opportunity to attach debugger");
                 syntax.DefineOptionList("codegenopt", ref _codegenOptions, "Define a codegen option");
                 syntax.DefineOptionList("rdxml", ref _rdXmlFilePaths, "RD.XML file(s) for compilation");
+                syntax.DefineOptionList("x", ref _illinkDescriptorsFilePaths, "ILLink descriptor file(s) for compilation");
```

Comment:
I'm not sure if we want to "waste" the `-x` option on this. Maybe `--linkdescriptor` would be better. I know il linker uses x, but we don't need to be compatible with the il linker command line.